feat: Editorial review for @scope docs (#30290)
mdn/content: files/en-us/web/css/_colon_scope/index.md

#### HTML

```html
<div class="light-scheme">
  <p>
    MDN contains lots of information about
    <a href="/en-US/docs/Web/HTML">HTML</a>,
    <a href="/en-US/docs/Web/CSS">CSS</a>, and
    <a href="/en-US/docs/Web/JavaScript">JavaScript</a>.
  </p>
</div>

<div class="dark-scheme">
  <p>
    MDN contains lots of information about
    <a href="/en-US/docs/Web/HTML">HTML</a>,
    <a href="/en-US/docs/Web/CSS">CSS</a>, and
    <a href="/en-US/docs/Web/JavaScript">JavaScript</a>.
  </p>
</div>
```

#### CSS

```css hidden
div {
  padding: 10px;
}
```

```css
@scope (.light-scheme) {
  :scope {
    background-color: plum;
  }

  a {
    color: darkmagenta;
  }
}

@scope (.dark-scheme) {
  :scope {
    background-color: darkmagenta;
    color: antiquewhite;
  }

  a {
    color: plum;
  }
}
```

#### Result

{{ EmbedLiveSample("Using :scope to style the scope root in a @scope block", "100%", 150) }}

### Using `:scope` in JavaScript

> This example demonstrates using the `:scope` pseudo-class in JavaScript. A situation where it can be useful is when you need to get a direct descendant of an already retrieved {{domxref("Element")}}.

Reviewer: ```suggestion
This example demonstrates using the `:scope` pseudo-class in JavaScript. This can be useful if you need to get a direct descendant of an already retrieved {{domxref("Element")}}.
```
